testsuite/src/test/java/org/jboss/da/test/communication/AproxTest.java

```diff
@@ -30,9 +30,9 @@ public static EnterpriseArchive createDeployment() {
 
     @Test
     public void testGetVersionsOfGA() throws CommunicationException {
-        GA ga = new GA("org.jboss.apiviz", "apiviz");
-        List<String> apivizTest = Arrays.asList(new String[] { "1.3.1.GA-redhat-1", "1.2.5.GA",
-                "1.3.0.GA", "1.3.1.GA-redhat-2", "1.3.2.GA" });
+        GA ga = new GA("org.jboss.ballroom", "ballroom");
+        List<String> apivizTest = Arrays.asList(new String[] { "1.3.0.Final-redhat-1",
+                "1.4.0.Final-redhat-1", "1.6.0.Final-redhat-1" });
         List<String> result = aproxConnector.getVersionsOfGA(ga);
         assertTrue(apivizTest.size() == result.size());
         assertTrue(apivizTest.containsAll(result));
```

Comment:
I'd suggest to use apivizTest.size() <= result.size() and switch the lists in contains all to prevent failing tests after the next release. It can serve as an intergration test. If we want more tests, then we can mock the AProx